### ISSUE TITLE
why does r_io need to know if we do debugging or not?

### ISSUE THREAD

**Issue by condret** [radare/radare2#4370]

**Comment by radare** [comment]
Already answered on irc

**Comment by ret2libc** [comment]
Oh wait :D that's a bold move! I'm not that familiar with RIO. Moreover, that's definitely not something I'll be able to do for 3.3.0.

**Comment by radare** [comment]
Yeah its not for 3.3, its probably better not to touch io again until the next release. Just asking if u had any comment on this

**Comment by radare** [comment]
ping @ret2libc

**Comment by ret2libc** [comment]
As said before I'm not familiar enough with r_io to answer such question or work on this
